be1-go/hub/standard_hub/mod_test.go

```diff
@@ -890,21 +891,21 @@ func Test_Handle_Answer(t *testing.T) {
 	})
 	require.Error(t, sock.err, "rpc message sent by a client should be a query")
 	sock.err = nil
-	require.False(t, queryState)
+	require.False(t, *hub.queries.GetQueryState(1))
```

Comment:
these calls really make me think it shouldn't be a pointer, we could just return bool + err :-)